modules/game_console/console.lua

```diff
@@ -374,6 +395,32 @@ function clear()
   end
 end
 
+function switchMode(newView)
+  if newView then
+    consolePanel:setImageColor('#ffffff88')  
+  else
+    consolePanel:setImageColor('white')  
+  end
+  --consolePanel:setDraggable(floating)
+  --consoleTabBar:setDraggable(floating)
+  --floatingMode = floating
+end
+
+function onDragEnter(widget, pos)
+  return floatingMode
+end
+
+function onDragMove(widget, pos, moved)
+  if not floatingMode then
+    return
+  end
+  -- update margin
+  return true
+end
+
+function onDragLeave(widget, pos)
+  return floatingMode
+end
```

Comment:
What does this functions do depending on the returned value? (I've no idea)

Reply:
yea i tried to add staff from otcv8 but i ended up abadoning it...